docs/nima/openapi/openapi.adoc

```diff
@@ -0,0 +1,128 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
There is content in this file still referring to SE, and at least some of the example code will change. 

We might want to strip this file down as with others so it is only a placeholder to be rewritten when the OpenAPI code goes in/is updated for 4.x.